src/catchup/DownloadApplyTxsWork.cpp

```diff
@@ -129,4 +131,11 @@ DownloadApplyTxsWork::onSuccess()
 {
     mLastApplied = mApp.getLedgerManager().getLastClosedLedgerHeader();
 }
+
+std::string
+DownloadApplyTxsWork::getStatus() const
+{
+    return fmt::format("{}: Checkpoint to queue: {}", getName(),
```

Comment:
I don't know how useful is it to expose `mCheckpointToQueue`, as it actually doesn't give us much information about the current state of application. Maybe what we want is something like: "last applied ledger is N, total checkpoints queued for application: M"?

Reply:
Changed to to return progress